File: Java/やさしいJava/6/sample2.java
import java.io.*;

class sample2{
	public static void main(String[] args)throws IOException{
		System.out.println("テストの点数を入力してください。(０で終了)");
		
		BufferedReader br=
		 new BufferedReader(new InputStreamReader(System.in));
		 
		 int i=0;
		 int sum=0;
		 while(i==0){
		 	String str = br.readLine();
		 	int res = Integer.parseInt(str);
		 	
		 	if(res != 0)
		 		sum+=res;
		 	
		 	else
		 		i++;
		 }
		 System.out.println("テストの点数の合計は"+sum+"点です。");
		}
}
		 
		 

File: C言語/やさしいC/Lesson5/Sample2.c
#include <stdio.h>
//switch文　これもjavaと同じ
int main(void) {
    int res;
    printf("整数を入力してください。\n");
    scanf("%d", &res);

    switch(res) {
        case 1:
            printf("入力値は1です。\n");
            break;
        case 2:
            printf("入力値は2です。\n");
            break;
        default:
            printf("入力値は1,2ではありません。\n");
            break;
    }
    return 0;
}